src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding-top: 32px;
+  padding-left: 16px;
+  padding-right: 16px;
+  align-items: center;
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+
+    object-fit: cover;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    width: 100%;
+    margin: 0;
+    margin-bottom: 4px;
+    color: $text-accent-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__number {
+    align-self: flex-start;
+    color: $text-secondary-color;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
it's a default value for this, remove it
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight
```suggestion
```